### DISCUSSION TITLE
Does winget update all 3rd party apps

### DISCUSSION THREAD

**Discussion by Pravekumar** [microsoft/winget-cli#2332] [General] [answered]
Hi, I wanted to check if winget updates all 3rd party apps or this is limited only to 3rd party apps downloaded through MS store. as the extensions are different, need some clarity on this so that we can start using winget to start patching 3rd party apps. 

Regards,
PK

**Comment by Masamune3210** [comment, marked as answer]
If it's in the manifest repo, it's supported! And if something isn't in there you think should, you can go through a quick process to get it added

**Reply by Pravekumar** [reply]
Thank you, I finished testing all the apps in my scope and it works fine except for the one which requires license. Just thinking if there is a way to automate the updates to be deployed on the machines with out users having to do anything .

**Reply by OfficialEsco** [reply]
In theory all Windows Computers should have WinGet installed, so you could easily write a script to run winget install / upgrade. (Should probably include a `Get-Command winget`)

WinGet is going to be implemented into InTune later, however until then its used as a Win32 bundle which these projects have looked into
* https://github.com/o-l-a-v/winget-intune-win32
* https://github.com/Romanitho/Winget-Install

If you want full control over the applications you can remove the Official Sources and then make your own WinGet Source.
